iceoryx_binding_c/README.md

```diff
@@ -14,16 +14,25 @@ Therefore, we have the following coding conventions exclusively in the C API.
  - C functions are using an abbreviation of the `ClassName` for instance `cn` and
       are named like `iox_cn_method_name` where the camelCase is converted into 
```

Comment:
I'm not to happy with the abbrevation `cn`. From my point of view this is very uncommon.

Reply:
This was already discussed in the first c binding pull request and there we decided to use snake case and abbreviations since this is the good coding style in C. @elBoberido  what is your opinion?

Reply:
the idea was to orient on a C API like POSIX, which also does this, e.g. `sem_post`. If we cannot find a good abbreviation and the name is not too long, then an exception from this pattern should be fine

Reply:
I changed the abbreviation example so that it is a little bit more understandable